specs/http-api/README.md

```diff
@@ -196,34 +208,65 @@ No authorization required. Offerings are publicly accessible
 # Exchanges
 An exchange is a series of linked tbDEX messages between Alice and a PFI for a single exchange. An exchange can be created by submitting an RFQ.
 
-## Submit RFQ
+## Create Exchange
 
 ### Endpoint
-`POST /exchanges/:exchange_id/rfq`
+`POST /exchanges/:exchange_id`
 
 ### Authentication
 Refer to [Signature Verification Section]() of the tbDEX spec  
 ### Authorization
-No Authorization required to submit an RFQ
+No Authorization required to create an exchange
 
 ### Request Body
+
+#### `CreateExchangeRequest`
+| field            | data type | required | description                                                                                       |
+| ---------------- | --------- | -------- | ------------------------------------------------------------------------------------------------- |
+| `rfq`           | object    | Y        | The request for quote                       |
+| `replyTo` | object    | N        | A string containing a valid URI where new messages from the PFI will be sent |
+
 > [!IMPORTANT]
 > See RFQ structure [here](../README.md#rfq-request-for-quote)
 
+```json
+{
+  "rfq": {
+    {
+      "metadata": {
+        "from": "did:key:z6Mks4N5XdrE6VieJsgH8SMSRavmTox74RqoroW7bZzBLQBi",
+        "to": "did:ex:pfi",
+        "kind": "rfq",
+        "id": "rfq_01ha835rhefwmagsknrrhvaa0k",
+        "exchangeId": "rfq_01ha835rhefwmagsknrrhvaa0k",
+        "createdAt": "2023-09-13T20:19:28.430Z"
+      },
+      "data": {
+        "offeringId": "abcd123",
+        "payinMethod": {
+          "kind": "DEBIT_CARD",
+          "paymentDetails": "<HASH_PRIVATE_PAYIN_METHOD_PAYMENT_DETAILS>"
+        },
+        // ...
+      }
+    }
+  },
+  "replyTo": "https://alice.wallet.com/events"
+}
+```
+
 ### Response
 | Status             | Body                  |
 | ------------------ | --------------------- |
 | `202: Accepted`    | N/A                   |
 | `400: Bad Request` | `{ errors: Error[] }` |
 
 ### Errors
-| Status | Description             |
-| ------ | ----------------------- |
-| 400    | Validation error(s)     |
-| 400    | Failed Signature Check  |
-| 404    | Exchange not found      |
-| 409    | RFQ already exists      |
-| 409    | Exchange already exists |
+| Status | Description                  |
+| ------ | -----------------------------|
+| 400    | Validation error(s)          |
+| 400    | Failed Signature Check       |
+| 409    | Exchange already exists      |
```

Comment:
do we want to still keep the 404 exchange not found?

Reply:
No, since this endpoint is what creates an exchange, there should not be an existing exchange with the same `exchange_id`